apps/nextjs-website/src/components/Hero.tsx

```diff
@@ -1,13 +1,11 @@
 'use client';
-import { Hero as HeroEC } from '@pagopa/pagopa-editorial-components';
-import { HeroProps } from '@pagopa/pagopa-editorial-components/dist/components/Hero/index';
-import { Stack, useTheme } from '@mui/material';
 import MarkdownRenderer from './MarkdownRenderer';
+import { Hero as HeroRC } from '@react-components';
+import { HeroProps } from '@react-components/types/Hero/Hero.types';
```

Comment:
```suggestion
import { HeroProps } from '@react-components/types';
```
Not a big fan of this long import.

I would add a `index.ts` file inside the `types` folder to export types in a cleaner way.

Much like the `index.ts` file inside of the `components` folder.

Doing that would make the import look like the suggestion above.